src/lib/env.js

```diff
@@ -40,21 +38,12 @@ export class Env {
     return parser.parse.call(parser, emit, data);
   }
 
-  _createEntries(lang, ast) {
-    const entries = Object.create(null);
-    const create = lang.src === 'qps' ?
-      createPseudoEntry : createEntry;
-
-    for (let i = 0, node; node = ast[i]; i++) {
-      const id = node.$i;
-      if (id in entries) {
-        this.emit('duplicateerror', new L10nError(
-         'Duplicate string "' + id + '" found in ' + lang.code, id, lang));
-      }
-      entries[id] = create(node, lang);
+  _createEntries(lang, entries) {
+    if (lang.src !== 'qps') {
+      return entries;
     }
 
-    return entries;
+    return entries.map(createPseudoEntry.bind(this, lang));
```

Comment:
umm, walkContent seems to take a single entry. Do you want me to turn it around?